AdvancedPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/NotificationsPlayerActivity.java

```diff
@@ -0,0 +1,242 @@
+            Log.d("Note", "Desired state is paused. Video should not be actively rendering while desired state is paused");
+            break;
+          default:
+            break;
+        }
+        break;
+      case OoyalaPlayer.BUFFER_CHANGED_NOTIFICATION_NAME:
+        break;
+
+      case OoyalaPlayer.SEEK_STARTED_NOTIFICATION_NAME:
+        Log.d("Note", "Seek Started");
+        break;
+      case OoyalaPlayer.SEEK_COMPLETED_NOTIFICATION_NAME:
+        Log.d("Note", "Seek Complete");
+        break;
+
+      case OoyalaPlayer.ERROR_NOTIFICATION_NAME:
+        Log.d("Note", "Playback Error. Code:" + player.getError().getCode() + ", Description: " + player.getError().getMessage());
+        break;
+
+      case OoyalaPlayer.CC_STYLING_CHANGED_NOTIFICATION_NAME:
+        break;
+
+      case OoyalaPlayer.AD_OVERLAY_NOTIFICATION_NAME:
+        break;
+      case OoyalaPlayer.AD_STARTED_NOTIFICATION_NAME:
+        break;
+      case OoyalaPlayer.AD_COMPLETED_NOTIFICATION_NAME:
+        break;
+      case OoyalaPlayer.AD_POD_STARTED_NOTIFICATION_NAME:
+        Log.d("Note", "An ad manager has taken control of the OoyalaPlayer");
+        break;
+      case OoyalaPlayer.AD_POD_COMPLETED_NOTIFICATION_NAME:
+        Log.d("Note", "An ad manager has returned control of the OoyalaPlayer");
+        break;
+      case OoyalaPlayer.AD_SKIPPED_NOTIFICATION_NAME:
+        break;
+      case OoyalaPlayer.AD_ERROR_NOTIFICATION_NAME:
+        Log.d("Note", "Ad Playback Error");
+        break;
+
+
+      // NOTE: Unused for BaseStreamPlayer. However, these are usable in ExoPlayer
+      case OoyalaPlayer.BUFFERING_STARTED_NOTIFICATION_NAME:
+        break;
+      // NOTE: Unused for BaseStreamPlayer. However, these are usable in ExoPlayer
+      case OoyalaPlayer.BUFFERING_COMPLETED_NOTIFICATION_NAME:
+        break;
+
+      // Deprecated: Used specifically for DRM notifications within SecurePlayer playback
+      case OoyalaPlayer.DRM_RIGHTS_ACQUISITION_STARTED_NOTIFICATION_NAME:
+        break;
+
+      // Deprecated: Used specifically for DRM notifications within SecurePlayer playback
+      case OoyalaPlayer.DRM_RIGHTS_ACQUISITION_COMPLETED_NOTIFICATION_NAME:
+        break;
+
+      case OoyalaPlayer.LIVE_CC_AVAILABILITY_CHANGED_NOTIFICATION_NAME:
+        break;
+      case OoyalaPlayer.LIVE_CC_CHANGED_NOTIFICATION_NAME:
+        break;
+      case OoyalaPlayer.CC_CHANGED_NOTIFICATION_NAME:
+        break;
+      case OoyalaPlayer.CLOSED_CAPTIONS_LANGUAGE_CHANGED_NAME:
+        break;
+    }
+
+    // Automation Hook: to write Notifications to a temporary file on the device/emulator
+    String text="Notification Received: " + arg1 + " - state: " + player.getState();
+    // Automation Hook: Write the event text along with event count to log file in sdcard if the log file exists
+    playbacklog.writeToSdcardLog(text);
+
+    //Generic notification handler
+    Log.d(TAG, "Notification Received: " + arg1 + " - state: " + player.getState());
```

Comment:
You could be reusing the `text` variable you defined previously here. It's identical :)